packages/xo-server-auth-ldap/src/index.js

```diff
@@ -172,12 +237,18 @@ class AuthLdap {
       base: searchBase,
       filter: searchFilter = DEFAULTS.filter,
       startTls = false,
+      groups,
+      uri,
+      users,
     } = conf
 
     this._credentials = credentials
+    this._serverUri = uri
     this._searchBase = searchBase
     this._searchFilter = searchFilter
     this._startTls = startTls
+    this._groupsConfig = groups
+    this._usersConfig = users
```

Comment:
Can `groups` (I don't think so) or `users` be undefined`?

If yes, be careful when using them.